### PR TITLE
Blog post transcript from lightning session

### PR DESCRIPTION
**Pull request by DavidHernandez**
@RachelBaram could you review this?

### CODE REVIEW

File: src/_posts/2017-05-11-transcript-why-drupal.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+*Do I have more privileges than the other person? If so, then I have to use those privileges to help them and to help the rest of the people who also lacks this privilege.*
+
+*Is there people abusing their privilege? If so, then I have to confront them, even if their abuse does not directly affects me or even if it helps me, to stop the abuse of the less privileged people.*
+
+*When we gain something before the other person, we need to ask ourselves why did we receive it. Was it because we had more merits, or was it because we have more privileges? We have to make the differences of privileges more visible so people are aware of those differences.*
+
+*But... How is all of this related to Drupal?*
+
+![Free software](/assets/images/posts/why-drupal/free-software.png)
```
Comment (RachelBaram) [inline src/_posts/2017-05-11-transcript-why-drupal.md]: @DavidHernandez   Before making the changes, I wanted to make sure it is correct.  I think these changes may make more sense in translation: "Free software" >  "Open source software" ;   "private software" > "closed source software" or "proprietary software."  Let me know if this is ok by you.

Comment (DavidHernandez) [inline src/_posts/2017-05-11-transcript-why-drupal.md]: @RachelBaram no. There is a difference between Open Source and Free Software: Open Source is only about the code being accessible. Free Software is about rights and liberties. There is a sentence to clarify what it means "Free Software": _Free as in Freedom, not as a beer_. For that reason, usually is said "Free / Libre Software". "Libre" is a spanish word that means "Free as in freedom".

**Comment by RachelBaram** [comment]
@DavidHernandez Done with review/edit.

**Comment by RachelBaram** [comment]
@IshaDakota Please review and merge.

**Comment by IshaDakota** [comment]
Great post @DavidHernandez !

**Pull request merged**